### PR TITLE
Added path to frame display in camera view

### PR DESCRIPTION
**Pull request by ALouis38**
It can be useful sometimes to have the full path of the frame displayed in the camera view available.
Maybe this should be an optional display?

### CODE REVIEW

**Comment by mleotta** [comment]
@mwoehlke-kitware This branch is short and should be an easy review.  Do you see any issues?  I agree that it might nice to have the option of hiding this to save screen real estate, but I'm willing to take it as is.

**Comment by mleotta** [comment]
Tested and works for me. 👍 

Waiting for a 👍  from @mwoehlke-kitware

**Comment by mleotta** [comment]
Actually, we should also add a release note about this in doc/rel_notes/0.9.0.txt before it is merged

File: gui/MainWindow.cxx [hunk 1]
```diff
@@ -454,6 +454,7 @@ void MainWindowPrivate::updateCameraView()
     return;
   }
 
+  this->UI.cameraView->setFrameName(cd.imagePath);
   // Show landmarks
```
Comment (mwoehlke-kitware) [inline gui/MainWindow.cxx]: Ack, please add a blank line after this!

File: gui/CameraView.h [hunk 2]
```diff
@@ -56,6 +56,8 @@ class CameraView : public QWidget
 
   void addFeatureTrack(kwiver::vital::track const&);
 
+  void setFrameName(QString frameName);
```
Comment (mwoehlke-kitware) [inline gui/CameraView.h]: This feels like it ought to be a slot... also, `QString const&`?

File: gui/CameraView.cxx [hunk 3]
```diff
@@ -607,6 +607,13 @@ void CameraView::addFeatureTrack(kwiver::vital::track const& track)
   d->updateFeatures(this);
 }
 
+void CameraView::setFrameName(QString frameName)
+{
+  QTE_D();
+
```
Comment (mwoehlke-kitware) [inline gui/CameraView.cxx]: superfluous :smile: (...which I wouldn't bother to note except for other more interesting nits)

**Comment by mwoehlke-kitware** [comment]
I think it's fine being always-on, but I think I'd prefer it at the bottom rather than the top :smile:. Also, any reason why the text shouldn't be selectable? That seems like an obvious convenience feature...

File: gui/CameraView.cxx [hunk 4]
```diff
@@ -607,6 +607,13 @@ void CameraView::addFeatureTrack(kwiver::vital::track const& track)
   d->updateFeatures(this);
 }
 
+void CameraView::setFrameName(QString frameName)
```
Comment (mwoehlke-kitware) [inline gui/CameraView.cxx]: Missing separator line... @ALouis38, can you please pay attention to maintaining style consistency with the files you are editing? Thanks!

**Pull request merged**